sample-app/src/main/java/com/liferay/analytics/client/android/sample/MainActivity.kt

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.analytics.client.android.sample
+
+import android.content.Intent
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.widget.Button
+import com.liferay.analytics.client.android.api.impl.IdentityClientImpl
+import com.liferay.analytics.client.android.forms.FormContext
+import com.liferay.analytics.client.android.forms.Forms
+
+/**
+ * @author Igor Matos
+ */
+class MainActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        val formContext = FormContext(
```

Comment:
I don't know if FormContext is the best name for this use case

Reply:
But you just use the id, right? And right now you are just storing the id and the name. FormContext is a concept used inside the portal but mobile developers don't understand it. You could do the same receiving the id and the name as parameters. Wouldn't work if you are going to pass other things in the future, that I don't know.

Reply:
Definitely, I think the class is important but I don't think it should be exposed as the API of the library (just used internally) if we only need the id and the name we could do 
```kotlin
Forms.formSubmitted('id', 'name'); 
```
And that way developers wouldn't need to learn something new.

ALTHOUGH it's true that if they want to send several events with the same form it's more convenient to have a constant with the id and name and reuse the constant. I'm not really happy with the name and reminds me of a God Object (like ServiceContext), I would prefer FormAttributes or something like that but it's basically the same...

Leave like it is, then :)

Reply:
Okay, I agree that its better rename it to FormsAttributes then